src/azure-cli-core/azure/cli/core/_util.py

```diff
@@ -176,6 +177,17 @@ def b64encode(s):
         return encoded.decode('latin-1')
 
 
+def b64_to_hex(s):
```

Comment:
Well, presumably any command module that can integrate with KeyVault will need these methods, so I see the logic of putting this in core.